File: app/src/main/java/com/example/megaloma/memopad/db/MemoDetail.java
package com.example.megaloma.memopad.db;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

@Entity(tableName = "Memo")
public class MemoDetail {
    @PrimaryKey(autoGenerate = true)
    public int id = 0;

    public String write_date;
    public String title;
    public String content;
    int fav;
}

File: app/src/main/java/com/example/megaloma/memopad/db/MemoDetail.kt
package com.example.megaloma.memopad.db

import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity(tableName = "Memo")
class MemoDetail {
    @PrimaryKey(autoGenerate = true)
    var id: Int = 0
    var write_date: String? = null
    var title: String? = null
    var content: String? = null
    var fav: Int = 0
}
